File: demo-vk-rest-scraping.java/src/main/java/demo/json/CountriesSerializer.java
package demo.json;

import com.google.gson.Gson;
import com.google.gson.GsonBuilder;
import demo.model.CountriesList;

class CountriesSerializer {

    private static final Gson GSON = new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();

    static CountriesList fromJson(String json) {
        return GSON.fromJson(json, CountriesList.class);
    }

    static String toJson(CountriesList countries) {
        return GSON.toJson(countries);
    }
}

File: demo-vk-rest-scraping.kotlin/src/main/kotlin/demo/json/CountriesSerializer.kt
package demo.json

import com.google.gson.Gson
import com.google.gson.GsonBuilder
import demo.model.CountriesList

internal object CountriesSerializer {

    private val GSON = GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create()

    fun fromJson(json: String): CountriesList {
        return GSON.fromJson(json, CountriesList::class.java)
    }

    fun toJson(countries: CountriesList): String {
        return GSON.toJson(countries)
    }
}
